packages/@remirror/core/src/builtins/__tests__/persistent-selection-extension.spec.ts

```diff
@@ -0,0 +1,60 @@
+import { extensionValidityTest, renderEditor } from 'jest-remirror';
+
+import { PersistentSelectionExtension } from '..';
+
+extensionValidityTest(PersistentSelectionExtension);
+
+describe('configuration', () => {
+  [
```

Comment:
Jest actually has it's own builtin `.each` test method. 

https://jestjs.io/docs/en/api#testeachtablename-fn-timeout

Reply:
I am starting to like jest :)